JS/Script.js

```diff
@@ -1,73 +1,100 @@
-const bookCollection = document.querySelector('#collection');
-const addBook = document.querySelector('#add-book');
-
-if (localStorage.getItem('Awesome Book Collection') === null) {
-  var library = [];
-} else {
-  var library = JSON.parse(localStorage.getItem('Awesome Book Collection'))
-}
+class Book {
+  constructor() {
+    this.bookCollection = document.querySelector('#collection');
+    this.addBook = document.querySelector('#add-book');
+    this.library = {};
+  }
 
-function storeEntryData(titleElement, authorElement) {
-  let entryObject = {};
-  entryObject.title = titleElement.value;
-  entryObject.author = authorElement.value;
-  library.push(entryObject);
-}
+  storeEntryData = (titleElement, authorElement) => {
+    const entryObject = {};
+    entryObject.title = titleElement.value;
+    entryObject.author = authorElement.value;
+    this.library.push(entryObject);
+  }
```

Comment:
- [OPTIONAL] I would advise you to use the `concat` method instead `push` method. The push method mutates arrays. Mutations may lead to unexpected and hard-to-debug issues, where data becomes incorrect somewhere, and you have no idea where it happens.